tools/external-exposure/load-balancer.yaml

```diff
@@ -6,8 +6,7 @@ metadata:
     app.kubernetes.io/instance: $DEPLOYMENT
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: neo4j
-    statefulset.kubernetes.io/pod-name: $DEPLOYMENT-neo4j-core-$IDX
-  name: $DEPLOYMENT-neo4j-external-$IDX
+  name: $DEPLOYMENT-neo4j-external
```

Comment:
same as comment above -- this change is unhelpful if you want to maintain the old exposure instructions at the same time as permitting the new behavior.

In general, if you remove the old stuff (which we agree is no longer the right way to do it) this will likely confuse people who are looking for the old instructions and haven't migrated to 4.3 yet.

Alternatively, you could retain both approaches in the code & documentation (version dependent) and then put into the docs when do each.  Either approach is OK, but the way I read this PR right now, you're maintaining the old documentation (-> externalexposure42.adoc) and breaking some of its dependencies in the repo.

If you choose not to keep the old artifacts around suitable to be used, then this is one of those where you might consider migration documentation for people going 4.2 -> 4.3 with SSR.

Reply:
Migration docs are a good idea.

This is basically an omission, thanks for catching, I'll move the "old" examples to a new folder `external-exposure-legacy` or something